web/gds-admin-ui/src/pages/app/details/ReviewNotes/EditReviewNote.js

```diff
@@ -53,4 +54,11 @@ function EditReviewNote({ note, handleCancelEditingClick, vaspId, setIsEditable
     )
 }
 
+EditReviewNote.propTypes = {
+    note: PropTypes.object.isRequired,
+    handleCancelEditingClick: PropTypes.func.isRequired,
+    vaspId: PropTypes.number.isRequired,
```

Comment:
Just to double-check to make sure, `vaspId` here should be number or string?

Reply:
it should be string, thank you